sunbird-mentoring/constants/routes.js

```diff
@@ -1522,6 +1514,665 @@ module.exports = {
 				path: '/mentoring/v1/availability/users',
 				type: 'GET',
 			},
+		},
+        {
+			sourceRoute: '/interface/v1/account/create',
+			type: 'POST',
+			inSequence: true,
+			orchestrated: true,
+			targetRoute: {
+				path: '/user/v1/account/create',
+				type: 'POST',
+				functionName: 'createUser',
+			},
+		},
+		{
+			sourceRoute: '/interface/v1/account/update',
+			type: 'PATCH',
+			inSequence: false,
+			orchestrated: true,
+			targetRoute: {
+				path: '/user/v1/user/update',
+				type: 'POST',
+				functionName: 'updateUser',
+			},
+		},
+		{
+			sourceRoute: '/interface/user-signup',
```

Comment:
@rakeshSgr do we need these routes in sunbird mentoring? Let's keep only the ones needed.